pyproject.toml

```
@@ -105,14 +105,14 @@ classifiers = [  # Optional
 # https://packaging.python.org/discussions/install-requires-vs-requirements/    # noqa
 # THIS PROJECT NEEDS MPI PRE-installed IN OUR LOCAL SYSTEM
 dependencies = [ # Optional
-  "piecewise_regression",
+ "piecewise_regression",

```

looks like there's a change here you weren't expecting to make
```suggestion
  "piecewise_regression",
```

---

fixed